src/components/ArticlesContainer.jsx

```diff
@@ -0,0 +1,50 @@
+import axios from "axios"
+import { useEffect, useState } from "react";
+import ArticlesCard from "./ArticlesCard";
+
+export default function ArticlesContainer() {
+    const [articles, setArticles] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+
+    useEffect(() => {
+        const fetchArticles = async () => {
+            try {
+                const res = await axios.get('https://nc-news-31tf.onrender.com/api/articles')
+                setArticles(res.data.articles)
+                setIsLoading(false)
+            }
+            catch (error) {
```

Comment:
I'd separate this out into an **api.js** file - this way you can set a base URL and use this for making API calls.